backpack/core/derivatives/lstm.py

```diff
@@ -349,14 +326,13 @@ def _weight_ih_l0_jac_t_mat_prod(
             module, mat, subsampling=subsampling
         )
 
+        N_axis = get_batch_axis(module, "input0")
```

Comment:
```suggestion
        N_axis, _ = self._get_batch_and_time_axes(module)
```
maybe it's better to use the method from the same class

Reply:
Agree. Also:
- [ ] call `get_batch_and_time_axes` from `get_batch_axis`

Reply:
I agree that it would be nice to change `get_batch_axis`.
However, this would lead to a cyclic import (`subsampling` imports `lstm.get_batch_and_time_axes` and `lstm` imports `subsampling.subsample`).
Solution would be to move `get_batch_axis`()` to a separate file (e.g. `batch_axis.py`). This would be a better sorting of functions but involves many files.

Reply:
Thanks for summing up the options. There would be a significant diff if we move `get_batch_axis`.